packages/e2e-test-utils-playwright/src/request-utils/menus.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Internal dependencies
+ */
+import type { RequestUtils } from './index';
+
+export interface MenuData {
+	title: string;
+	content: string;
+}
+export interface Menu {
```

Comment:
Should we use the name Navigation?

Reply:
`navigationMenus`?

Reply:
I chose menu because it is used both for classic and navigation menus, but I think we can go with NavigationMenu too.